test/SparqlExpressionTest.cpp

```diff
@@ -894,3 +893,24 @@ TEST(SparqlExpression, replaceChildThrowsIfOutOfRange) {
   ASSERT_THROW(expr.replaceChild(1, std::move(exprToSubstitute)),
                ad_utility::Exception);
 }
+
+// ______________________________________________________________________________
+TEST(SparqlExpression, isAggregateAndIsDistinct) {
+  sparqlExpression::IdExpression idExpr(ValueId::makeFromInt(42));
+
+  ASSERT_FALSE(idExpr.isAggregate());
+  ASSERT_THROW(idExpr.isDistinct(), ad_utility::Exception);
+
+  Variable varX("?x");
+  sparqlExpression::detail::AvgExpression avgExpression(
+      false, std::make_unique<VariableExpression>(varX));
+
+  ASSERT_TRUE(avgExpression.isAggregate());
+  ASSERT_FALSE(avgExpression.isDistinct());
+
+  sparqlExpression::detail::AvgExpression avgDistinctExpression(
+      true, std::make_unique<VariableExpression>(varX));
+
+  ASSERT_TRUE(avgDistinctExpression.isAggregate());
+  ASSERT_TRUE(avgDistinctExpression.isDistinct());
```

Comment:
Technically you can and maybe should add another test for the `GROUP_CONCAT` to boost the coverage there.